File: SDKs/amazonaws.com/rds/2014-10-31/python/src/sdk/models/shared/range.py
from dataclasses import dataclass, field
from typing import Optional


@dataclass
class Range:
    r"""Range
    A range of integer values.
    """
    
    from_: Optional[int] = field(default=None)
    step: Optional[int] = field(default=None)
    to: Optional[int] = field(default=None)
    

File: SDKs/amazonaws.com/rds/2014-10-31/typescript/src/sdk/models/shared/range.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// Range
/** 
 * A range of integer values.
**/
export class Range extends SpeakeasyBase {
  @SpeakeasyMetadata()
  from?: number;

  @SpeakeasyMetadata()
  step?: number;

  @SpeakeasyMetadata()
  to?: number;
}
